pkg/api/controller_test.go

```diff
@@ -215,6 +215,24 @@ func TestController_GetRepoHandler(t *testing.T) {
 	})
 }
 
+	t.Run("use same storage namespace twice with bare repo", func(t *testing.T) {
```

Comment:
That's a great test - but note you could have reused the code of "use same storage namespace twice" and parameterize the test.
Look at the example of catalog_test.go:564 as reference

Reply:
cool, thanks for the feedback. I did that and added the parameter (isBareRepo=true\false) into the subtest name, thought it might be helpful.